wipac_dev_tools/logging_tools.py

```diff
@@ -0,0 +1,74 @@
+"""Common tools to supplement/assist the standard logging package."""
+
+import argparse
+import logging
+from typing import List, Optional, Union
+
+
+def log_argparse_args(
+    args: argparse.Namespace,
+    logger: Optional[Union[str, logging.Logger]] = None,
+    level: str = "warning",
```

Comment:
If you feel like overkill...

https://adamj.eu/tech/2021/07/09/python-type-hints-how-to-use-typing-literal/

https://docs.python.org/3/library/logging.html#levels

the type could be a literal list of acceptable log levels.

I think `"WARNING"` (uppercase) would be better, but I feel pedantic even suggesting it. :smile:

Maybe make it match the default on Line 40?

Reply:
I like that. It's a nice lighter weight option to enums :thinking: